plexapi/base.py

```diff
@@ -142,19 +142,21 @@ def fetchItem(self, ekey, cls=None, **kwargs):
         clsname = cls.__name__ if cls else 'None'
         raise NotFound('Unable to find elem: cls=%s, attrs=%s' % (clsname, kwargs))
 
-    def fetchItems(self, ekey, cls=None, **kwargs):
+    def fetchItems(self, ekey, cls=None, container_start=None, container_size=None, **kwargs):
```

Comment:
How is it possible now to pass `container_start` and `container_size` through e.g. `LibrarySection.all()`? Before it was possible through `**kwargs` but now it would have to be passed through explicitly which is currently not the case.

Reply:
Kwargs can replace keyword arguments. We still test for this.